corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -472,7 +472,7 @@ def get_datum_meta_module(self, module, use_filter=False):
             ))
 
             if module_offers_search(detail_module) and detail_module.search_config.data_registry:
-                datums.append(self.get_data_registry_query_datum(datum, detail_module))
+                datums.extend(self.get_data_registry_query_datum(datum, detail_module))
```

Comment:
Renaming to `get_data_registry_query_datums` would be a little clearer.

Reply:
9145e15cf6c